tests/BigBlueButtonTest.php

```diff
@@ -33,6 +33,7 @@
 use BigBlueButton\TestServices\EnvLoader;
 use BigBlueButton\TestServices\Fixtures;
 use BigBlueButton\TestServices\ParamsIterator;
+use Tracy\Debugger;
```

Comment:
If you can please run `cs-fix` and it will remove this unused import.

Reply:
done